### PR TITLE
Print actual labels without ellipsis

### LINKED ISSUE #878: 'print label' displays ... wildcard
```
$ buildozer --version
buildozer version: 3.4.0 
buildozer scm revision: b1667ff58f714d13c2bba6823d6c52214705508f
$ buildozer 'print label' '//...:*'
//...:go_default_library
//...:go_default_library
//...:go_default_test
//...:copy_LICENSE
//...:buildifier_launcher
//...:buildifier
//...:integration_test
//...:build
//...:go_default_library
//...:go_default_test
//...:go_default_library
//...:buildozer
//...:buildozer_test
//...:buildozer-darwin
//...:buildozer-linux
//...:buildozer-windows
//...:go_default_library
//...:unused_deps
//...:unused_deps-darwin
//...:unused_deps-linux
//...:unused_deps-windows
//...:jar_manifest_test
//...:go_default_test
//...:go_default_library
//...:buildtools
//...:windows
//...:warnings
//...:tests
//...:gazelle
//...:buildifier
//...:go_default_library
//...:go_default_test
//...:go_default_library
//...:buildifier2
//...:build_proto
//...:blaze_query_proto
//...:go_default_library
//...:api_proto
//...:api_proto_proto
//...:api_proto_go_proto
//...:go_default_library
//...:testutils
//...:deps_proto
//...:blaze_deps_proto
//...:go_default_library
//...:go_default_library
//...:generatetables
//...:go_default_library
//...:go_default_test
//...:go_default_library
//...:go_default_test
//...:go_default_library
//...:bzlenv_test
//...:go_default_test
//...:go_default_library
//...:go_default_library
//...:go_default_test
//...:copy_LICENSE
//...:compile_ts
//...:buildozer_launcher
//...:buildozer
//...:integration_test
//...:extra_actions_base_proto
//...:blaze_proto
//...:go_default_library
//...:buildifier
//...:buildifier-darwin
//...:buildifier-linux
//...:buildifier-windows
//...:buildifier_integration_test
//...:go_default_library
//...:go_default_library
//...:lang
//...:generateTablesFile
//...:go_default_library
```
I would expect the actual packages to be expanded in the output.

### PR DESCRIPTION
**Pull request by vladmos**
Buildifier's `print` command prints labels using the input which may contain an ellipsis (`...`). It should use the actual BUILD file location instead.

Some ultility functions have been moved from `buildifier/utils` (which loads other packages) to `utils` (which shouldn't load any other packages) to avoid circular dependencies.

Fixes #878

### CODE REVIEW

**Review by laurentlb — COMMENTED** [review]
`utils` is not a descriptive name, it's hard to know what to expect in there.

I'm confused as to why there's both github.com/bazelbuild/buildtools/buildifier/utils and github.com/bazelbuild/buildtools/utils.

**Comment by vladmos** [comment]
I originally wanted to create a package `utils` for any low-level utility funcitons (that do not require other packages), that's why it doesn't have a descriptive name. But perhaps I can move this methods to `wspace` as they all have something to do with file paths and the location of WORKSPACE in them.

**Comment by laurentlb** [comment]
Using wspace sounds good to me

**Comment by vladmos** [comment]
Merged `utils` with `wspace`, also refactored them as there were methods that do roughly the same thing.

File: buildozer/buildozer_test.sh [hunk 1]
```diff
@@ -1038,6 +1044,21 @@ java_library(name = "b")'
...
+  echo 'java_library(name = "foo")' > "ellipsis_test/foo/BUILD"
+  echo 'java_library(name = "foobar"); java_library(name = "bar");' > "ellipsis_test/foo/bar/BUILD"
+
+  in='package()
+java_library(name = "b")'
+  run "$in" 'print label' '//ellipsis_test/...:*'
+  assert_output_any_order '//ellipsis_test:test
+//ellipsis_test/foo:foo
+//ellipsis_test/foo/bar:foobar
+//ellipsis_test/foo/bar:bar'
```
[the remark this answers is not in the document]
Comment (vladmos) [reply buildozer/buildozer_test.sh]: Fixed.

**Review by laurentlb — APPROVED** [review]
lgtm

**Pull request merged**